covalent/_results_manager/results_manager.py

```diff
@@ -155,13 +150,12 @@ def _get_result_from_dispatcher(
     adapter = HTTPAdapter(max_retries=5)
     http = requests.Session()
     http.mount("http://", adapter)
-    url = "http://" + dispatcher + "/api/result/" + dispatch_id
-    response = http.get(url, params={"wait": wait, "status_only": status_only}, timeout=2)
+    url = f"http://{dispatcher}/api/result/{dispatch_id}"
+    response = http.get(url, params={"wait": wait, "status_only": status_only}, timeout=60)
```

Comment:
are you sure this is the right place for `timeout` and not `HTTPAdapter`?

Reply:
`timeout` is [a parameter](https://requests.readthedocs.io/en/latest/api/) of `Request`, not `HTTPAdapter`

Reply:
I'm actually not sure if `timeout` is even appropriate here (what if a dispatch really does take a really long time?), but I think that's outside the scope of this issue.